app/service/document.py

```diff
@@ -97,9 +97,14 @@ def update(
     # TODO: implement changing of a document's organisation
```

Comment:
Do we still need this TODO?

Reply:
Yeah - good to leave them in to process them all at once